src/Hyperwallet/Util/ApiClient.php

```diff
@@ -180,4 +182,18 @@ private function doRequest($method, $url, array $urlParams, array $options) {
         }
     }
 
+    /**
+     * Checks whether Content-Type header is valid in response
+     *
+     * @param string $response Response to be checked
+     *
+     * @throws HyperwalletException
+     */
+    private function checkResponseHeaderContentType($response) {
+        $contentTypes = $response->getHeader('Content-Type');
+        if (empty($contentTypes) || ($contentTypes[0] != 'application/json' && $contentTypes[0] != 'application/jose+json')) {
```

Comment:
This logic is not handling properly what the requirements says:
```
Given an HTTP Request with either no Accept Header or an Accept Header equal to "application/json"
When an HTTP Response is received containing a Content-Type Header not equal to "application/json"
Then an exception (Error in Node, HyperwalletException in Java and PHP, HyperwalletAPIException in Python) should be thrown with the message "Invalid Content-Type specified in Response Header"
```
and
```
Given an HTTP Request with an Accept Header equal to "application/jose+json"
When an HTTP Response is received containing a Content-Type Header not equal to "application/jose+json" or “application/json”
Then an exception (Error in Node, HyperwalletException in Java and PHP, HyperwalletAPIException in Python) should be thrown with the message "Invalid Content-Type specified in Response Header"
```
You are not considering what was defined on `request.header['Accept']`

Reply:
I think this is `response` object, I added the comment above to so that the code will be changed appropriately and supply the required requirements based on this ticket

Reply:
Fixed